src/mkdocstrings/handlers/base.py

```diff
@@ -314,6 +324,7 @@ def __init__(self, config: dict) -> None:
         """
         self._config = config
         self._handlers: Dict[str, BaseHandler] = {}
+        self.inventory: Inventory = Inventory()
```

Comment:
Project and version should be set here when instantiating the inventory.

Reply:
We'll only set the project name for now, using `site_name`. We'll see later for the version.